internal/model/installation.go

```diff
@@ -6,23 +6,25 @@ import (
 )
 
 const (
-	// InstallationStateCreationRequested is a installation in the process of being created.
```

Comment:
🤦‍♂

Reply:
Not a big deal! I am sure it was just a minor copy+pasta error.